acctest/main_test.go

```diff
@@ -83,7 +83,24 @@ func TestShouldFailIfServerIsNotListening(t *testing.T) {
 	// then
 	assert.Equal(t, 127, exitcode)
 	assert.Empty(t, stdout)
-	assert.Contains(t, stderr, "error")
+	assert.Contains(t, stderr, "connection refused")
+}
+
+func TestShouldFailIfServerDoesNotImplementHealthCheckProtocol(t *testing.T) {
+	// given
+	srv, err := StartEmptyServer(port)
+	if err != nil {
+		log.Fatalf("can't start stub server: %v", err)
+	}
+	defer srv.GracefulStop()
+
+	// when
+	stdout, stderr, exitcode := runBin(t, stubSrvAddr)
+
+	// then
+	assert.Equal(t, 127, exitcode)
+	assert.Empty(t, stdout)
+	assert.Equal(t, stderr, "rpc error: server doesn't implement gRPC health-checking protocol\n")
```

Comment:
Should we mention concrete version this utility supports? It's `grpc.health.v1` after all.

Reply:
I wouldn't worry about it, until something else than v1 is released (which, I bet, will likely never happen).  After all, it is just a human-readable message and we can change it anytime.

If someone is about to rely on exact wording of an error message, I'd suggest then to rely on exit code instead. We can introduce new ones if needed later